modules/nf-core/mobster/templates/main_script.R

```diff
@@ -58,11 +57,17 @@ if ( grepl(".rds\$", tolower("$rds_join")) ) {
 
 ## Function to run a single mobster fit
 run_mobster_fit = function(joint_table, descr) {
+    purity = unique(joint_table[["purity"]]) %>% as.numeric()
```

Comment:
Why are you doing this? What happens if there are more than one?

Reply:
Up to you, if it is never going to be a problem then you don't need one, but it doesn't hurt to add a line that throws a clear error when something goes wrong